.gitignore

```diff
@@ -0,0 +1,24 @@
+#################
+## Visual Studio
+#################
+
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# User-specific files
+*.suo
+*.user
+.vs/
+
+
+# Build results
+linker/obj
+linker/bin
+
+
+# Mac crap
```

Comment:
Haha sorry.  Saw that coming.  Someone else here noticed that in the PR and got a good laugh.

In my defense, I didn't write that.   That's part of the "Default gitignore file" that Github Desktop has a menu item to generate.